tests/python_package_test/test_sklearn.py

```diff
@@ -1,30 +1,35 @@
 # coding: utf-8
 import itertools
-import joblib
 import math
 import os
 
-import lightgbm as lgb
+import joblib
 import numpy as np
 import pytest
 from pkg_resources import parse_version
 from sklearn import __version__ as sk_version
 from sklearn.base import clone
 from sklearn.datasets import load_svmlight_file, make_multilabel_classification
-from sklearn.utils.estimator_checks import check_parameters_default_constructible
 from sklearn.metrics import log_loss, mean_squared_error
-from sklearn.model_selection import GridSearchCV, RandomizedSearchCV, train_test_split
-from sklearn.multioutput import (MultiOutputClassifier, ClassifierChain, MultiOutputRegressor,
-                                 RegressorChain)
+from sklearn.model_selection import (GridSearchCV, RandomizedSearchCV,
+                                     train_test_split)
+from sklearn.multioutput import (ClassifierChain, MultiOutputClassifier,
+                                 MultiOutputRegressor, RegressorChain)
+from sklearn.utils.estimator_checks import \
+    check_parameters_default_constructible
```

Comment:
We don't use controversial rule of 80 char line length limit in LightGBM.
```suggestion
from sklearn.utils.estimator_checks import check_parameters_default_constructible
```

Reply:
If this change was made by `isort` and not by `black`, we can mark it `# isort:skip`.

Reply:
Never mind! I created #3990.